src/lib/Handshake.js

```diff
@@ -97,7 +97,6 @@ class Handshake {
   _socket: Socket;
   _isSendingHello: boolean = false;
   _deviceID: string;
-  _pendingBuffers: Array<Buffer> = [];
```

Comment:
we never push actual buffers to this array, but map the array to routeMessage on device's startup..this doesn't make any sense..so I removed it.